e2e/fedora.yaml

```diff
@@ -0,0 +1,221 @@
+      # 🟢 Builtin default: "fscache" for non-writable mounts, "mmap" for writable mounts
+      cache: null
+  - location: "/tmp/lima"
+    # 🟢 Builtin default: false
+    # 🔵 This file: true (only for "/tmp/lima")
+    writable: true
+
+# Mount type for above mounts, such as "reverse-sshfs" (from sshocker) or "9p" (EXPERIMENTAL, from QEMU’s virtio-9p-pci, aka virtfs)
+# 🟢 Builtin default: "reverse-sshfs"
+mountType: reverse-sshfs
+
+ssh:
+  # A localhost port of the host. Forwarded to port 22 of the guest.
+  # 🟢 Builtin default: 0 (automatically assigned to a free port)
+  # NOTE: when the instance name is "default", the builtin default value is set to
+  # 60022 for backward compatibility.
+  localPort: 0
+  # Load ~/.ssh/*.pub in addition to $LIMA_HOME/_config/user.pub .
+  # This option is useful when you want to use other SSH-based
+  # applications such as rsync with the Lima instance.
+  # If you have an insecure key under ~/.ssh, do not use this option.
+  # 🟢 Builtin default: true
+  loadDotSSHPubKeys: false
+  # Forward ssh agent into the instance.
+  # 🟢 Builtin default: false
+  forwardAgent: null
+  # Forward X11 into the instance
+  # 🟢 Builtin default: false
+  forwardX11: null
+  # Trust forwarded X11 clients
+  # 🟢 Builtin default: false
+  forwardX11Trusted: null
+
+# ===================================================================== #
+# ADVANCED CONFIGURATION
+# ===================================================================== #
+
+containerd:
+  # Enable system-wide (aka rootful)  containerd and its dependencies (BuildKit, Stargz Snapshotter)
+  # 🟢 Builtin default: false
+  system: true
+  # Enable user-scoped (aka rootless) containerd and its dependencies
+  # 🟢 Builtin default: true
+  user: false
+#  # Override containerd archive
+#  # 🟢 Builtin default: hard-coded URL with hard-coded digest (see the output of `limactl info | jq .defaultTemplate.containerd.archives`)
+#  archives:
+#  - location: "~/Downloads/nerdctl-full-X.Y.Z-linux-amd64.tar.gz"
+#    arch: "x86_64"
+#    digest: "sha256:..."
+
+# Provisioning scripts need to be idempotent because they might be called
+# multiple times, e.g. when the host VM is being restarted.
+# 🟢 Builtin default: null
+provision:
+  # Install packages needed for QEMU user-mode emulation
+  - mode: system
+    script: |
+      #!/bin/bash
+      dnf install -y --setopt=install_weak_deps=False qemu-user-static-aarch64 qemu-user-static-arm qemu-user-static-x86
+  - mode: boot
+    script: |
+      systemctl stop NetworkManager-wait-online.service
+      systemctl reset-failed NetworkManager-wait-online.service
+      systemctl mask NetworkManager-wait-online.service
+  # # `user` is executed without the root privilege
+  - mode: user
+    script: |
+      #!/bin/bash
+      
+      # Enable SSHing into the VM as root (e.g., in `nerdctlConfigApplier.Apply`).
+      sudo cp ~/.ssh/authorized_keys /root/.ssh/
+      sudo chown $USER /mnt/lima-finch
```

Comment:
Removed lines related to persistent volume and specific configurations for finch. The tests don't have to run specifically against `finch`, the tests will ensure that the dependencies distributed by finch-core can work against lima based system, therefore the subject for our test is `limactl`. I also added a workflow to update the reference to the dependencies in the Makefile so that the e2e tests will run against the most recent set of built dependencies. Added more details in the description.